### PR TITLE
Add crop and compare image method in TizenDriver

### PR DESCRIPTION
**Pull request by yourina**
## Change list

Add crop and compare image method in TizenDriver
 
## Types of changes

What types of changes are you proposing/introducing to .NET client?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Details

Please provide more details about changes if it is necessary. If there are new features you can provide code samples which show the way they
work and possible use cases. Also you can create [gists](https://gist.github.com) with pasted C# code samples or put them here using markdown. 
About markdown please read [Mastering markdown](https://guides.github.com/features/mastering-markdown/) and [Writing on GitHub](https://help.github.com/categories/writing-on-github/)

### CODE REVIEW

File: appium-dotnet-driver/Appium/Tizen/TizenDriver.cs [hunk 1]
```diff
@@ -82,6 +85,29 @@ public TizenDriver(AppiumLocalService service, DesiredCapabilities desiredCapabi
         {
         }
 
+        public bool CompareToImages(Image origianalImage, Image compareImage)
```
Comment (rookiejava) [inline appium-dotnet-driver/Appium/Tizen/TizenDriver.cs:88]: API signature looks wired for me. 
What about `CompareImages(Image sourceImage, Image targetImage)` instead?

File: appium-dotnet-driver/Appium/Tizen/TizenDriver.cs [hunk 2]
```diff
@@ -82,6 +85,29 @@ public TizenDriver(AppiumLocalService service, DesiredCapabilities desiredCapabi
...
+        {
+            MemoryStream originalImageStream = new MemoryStream();
+            MemoryStream compareImageStream = new MemoryStream();
+
+            origianalImage.Save(originalImageStream, origianalImage.RawFormat);
```
Comment (rookiejava) [inline appium-dotnet-driver/Appium/Tizen/TizenDriver.cs:93]: need to check wheter image is null or not first.

File: appium-dotnet-driver/Appium/Tizen/TizenDriver.cs [hunk 3]
```diff
@@ -82,6 +85,29 @@ public TizenDriver(AppiumLocalService service, DesiredCapabilities desiredCapabi
...
+            compareImage.Save(compareImageStream, compareImage.RawFormat);
+
+            return string.Equals(Convert.ToBase64String(originalImageStream.ToArray()), Convert.ToBase64String(compareImageStream.ToArray()));
```
Comment (rookiejava) [inline appium-dotnet-driver/Appium/Tizen/TizenDriver.cs:96]: couldn't we optimzing it?

File: appium-dotnet-driver/Appium/Tizen/TizenDriver.cs [hunk 4]
```diff
@@ -82,6 +85,29 @@ public TizenDriver(AppiumLocalService service, DesiredCapabilities desiredCapabi
...
+        }
+
+        public Image CropAndConvertImage(Screenshot screenshot, Rectangle rectangle)
```
Comment (rookiejava) [inline appium-dotnet-driver/Appium/Tizen/TizenDriver.cs:99]: need to rename more clear something like `TransformScreenshot()` or `ConvertScreenshotToImage()`

File: appium-dotnet-driver/Appium/Tizen/TizenDriver.cs [hunk 5]
```diff
@@ -82,6 +85,29 @@ public TizenDriver(AppiumLocalService service, DesiredCapabilities desiredCapabi
...
+        {
+            MemoryStream imageStream = new MemoryStream(screenshot.AsByteArray);
```
Comment (rookiejava) [inline appium-dotnet-driver/Appium/Tizen/TizenDriver.cs:101]: who can gurantee the input is valid here?

File: appium-dotnet-driver/Appium/Tizen/TizenDriver.cs [hunk 6]
```diff
@@ -82,6 +85,29 @@ public TizenDriver(AppiumLocalService service, DesiredCapabilities desiredCapabi
...
+            Image screenshotImage = Image.FromStream(imageStream);
+
```
Comment (rookiejava) [inline appium-dotnet-driver/Appium/Tizen/TizenDriver.cs:103]: no need spacing between lines.

**Review by rookiejava — CHANGES_REQUESTED** [review]

File: appium-dotnet-driver/Appium/Tizen/TizenDriver.cs (hunk 2 above)
Comment (rookiejava) [reply appium-dotnet-driver/Appium/Tizen/TizenDriver.cs:93]: fix typo as well

**Comment by yourina** [comment]
@flyofsky  Could you check these comments and make new PR applied the cooment of @rookiejava? 
Thank you in advance.

**Pull request closed without merging**